include/seqan3/alignment/configuration/all.hpp

```diff
@@ -18,6 +18,7 @@
 #include <seqan3/alignment/configuration/align_config_edit.hpp>
 #include <seqan3/alignment/configuration/align_config_gap.hpp>
 #include <seqan3/alignment/configuration/align_config_max_error.hpp>
+#include <seqan3/alignment/configuration/align_config_parallel.hpp>
```

Comment:
I only recently learned that the includes have to be sorted lexicographically. So this include belongs under ... config_mode